<launch>
    <arg name="image_src" default="/usb_cam/image_raw" />
    <arg name="camera_info_src" default="/camera_info" />
    <arg name="calibration_file" default="$(find hdmap)/param/zed_lidar_camera_1080p.yaml"/>
    <arg name="target_frame" default="laser"/>
    <arg name="camera_frame" default="usb_cam"/>
    <arg name="projection_matrix_topic" default="/projection_matrix"/>

    <node pkg="hdmap" type="calibration_pub" name="calibration_pub" output="screen" >      
        <param name="register_lidar2camera_tf" type="bool" value="true"/>
        <param name="publish_extrinsic_mat" type="bool" value="true"/>
        <param name="publish_camera_info" type="bool" value="true"/>
        <param name="image_topic" value="$(arg image_src)"/>
        <param name="camera_info_src" value="$(arg camera_info_src)"/>
        <param name="calibration_file" value="$(arg calibration_file)"/>
        <param name="projection_matrix_topic" value="$(arg projection_matrix_topic)"/>
        <param name="target_frame" type="str" value="$(arg target_frame)"/>
        <param name="camera_frame" type="str" value="$(arg camera_frame)"/> 
        <param name="if_autoware_calibration" type="bool" value="true"/>
    </node>
</launch>
